tools/pr-review-advisor/local-review.mts

```
@@ -0,0 +1,298 @@
+          `Local review failed during ${stage} for specialist ${specialist.interest}`,
+          { cause: error },
+        );
+        throw specialistFailure;
+      } finally {
+        try {
+          if (activeEnvironment === env) {
+            activeEnvironment = undefined;
+            lifecycle.remove(env);
+          }
+        } catch (cleanupError) {
+          if (specialistFailure === undefined) throw cleanupError;
```

Biome (2.5.7)

[error] Unsafe usage of 'throw'.

(lint/correctness/noUnsafeFinally)